src/ESPiLight.cpp

```diff
@@ -273,14 +273,14 @@ int ESPiLight::createPulseTrain(uint16_t *pulses, const String &protocol_id,
 
 int ESPiLight::parsePulseTrain(uint16_t *pulses, int length) {
   int matches = 0;
-  struct protocol_t *protocol = NULL;
+  struct protocol_t *protocol = nullptr;
   struct protocols_t *pnode = used_protocols;
 
-  // Serial.println("piLightParsePulseTrain start");
-  while ((pnode != NULL) && (_callback != NULL)) {
+  // DebugLn("piLightParsePulseTrain start");
```

Comment:
This is wrong for this commit. Please move.

Reply:
I don't think its worth rebasing the whole thing again, as its a comment anyway. It does not break the build and the needed define comes in the same PR.

Reply:
I like to have a clean history. I can do the rebasing for you, but this will break your signatures.

Reply:
Feel free :)

Maybe we should consider removing commented code in general.

Reply:
Yes, you are right.